torch/csrc/jit/frontend/sugared_value.cpp

```diff
@@ -122,6 +122,8 @@ std::shared_ptr<SugaredValue> SimpleValue::attr(
            {"T", "prim"},
            {"ndim", "prim"},
            {"name", "prim"},
+           {"real", "prim"},
+           {"imag", "prim"},
```

Comment:
```suggestion
           {"real", "aten"},
           {"imag", "aten"},
```

I found these in `native_functions.yaml`, let's try using them instead of defining our own:
```
- func: real(Tensor(a) self) -> Tensor(a)
  variants: function

- func: imag(Tensor(a) self) -> Tensor(a)
  variants: function
```